recipes/opencl-icd-loader/all/conanfile.py

```diff
@@ -1,18 +1,19 @@
 from conans import ConanFile, CMake, tools
 import os
 
+required_conan_version = ">=1.33.0" 
 
 class OpenclIcdLoaderConan(ConanFile):
     name = "opencl-icd-loader"
     description = "OpenCL ICD Loader."
     license = "Apache-2.0"
-    topics = ("conan", "opencl-icd-loader", "opencl", "khronos", "parallel", "icd-loader")
+    topics = ("opencl-icd-loader", "opencl", "khronos", "parallel", "icd-loader")
     homepage = "https://github.com/KhronosGroup/OpenCL-ICD-Loader"
     url = "https://github.com/conan-io/conan-center-index"
 
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
+    options = {"shared": [True, False], "fPIC": [True, False], "disable_openclon12": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "disable_openclon12": False}
```

Comment:
please, in `config_options()` remove this option if not windows, and inject it in `_configure_cmake()` if windows.

https://github.com/KhronosGroup/OpenCL-ICD-Loader/blob/4e65bd5db0a0a87637fddc081a70d537fc2a9e70/CMakeLists.txt#L94-L100

Reply:
Commited